flink-connectors/flink-connector-kinesis/src/test/resources/log4j2-test.properties

```diff
@@ -0,0 +1,29 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+# Set root logger level to OFF to not flood build logs
+# set manually to INFO for debugging purposes
+rootLogger.level = OFF
```

Comment:
is this new logging file related to the PR?

Reply:
also in `flink-table/flink-table-api-java-bridge`

Reply:
adding the missing logging file sounds good
my concern was more about the fact that adding of the logging file is unrelated to the problem at hand.
asaik, we usually introduce such changes as separate hotfix commits

Reply:
Okay, I'll separate this out into a hotfix commit before merging.